plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Marketing.php

```diff
@@ -26,13 +26,14 @@ public function __construct( $task_list ) {
 	public function on_activated_plugin( $plugin ) {
 		$plugin_basename = basename( plugin_basename( $plugin ), '.php' );
 
-		if ( $plugin_basename === 'multichannel-by-cedcommerce' &&
-			$this->task_list->visible &&
-			! $this->task_list->is_hidden() &&
-			! $this->is_complete()
-		) {
-			$this->mark_actioned();
-		}
+		// Example: How to mark the marketing task as complete when a specific plugin is activated.
+		// if ( $plugin_basename === 'pluginbasename-slug' &&
+		//     $this->task_list->visible &&
+		//     ! $this->task_list->is_hidden() &&
+		//     ! $this->is_complete()
+		// ) {
+		//     $this->mark_actioned();
+		// }
```

Comment:
Could remove all the logic (including the `on_activated_plugin` callback on activation) since there is nothing being done here, but I figured it's useful for future updates. Not a strong opinion though.